tensorflow_addons/activations/tests/snake_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import pytest
+
+import numpy as np
+
+from tensorflow_addons.activations import snake
+
```

Comment:
nit: remove this empty line

Reply:
Sure. The reason I was doing so is I was trying to somehow group similar imports since there are no strict guidelines on how to order imports, nor other layers and activations adhere some consistent pattern.